v2.1/how-to/maintain/identify-slow-queries.md

```diff
@@ -257,3 +288,21 @@ admin show slow top all 5
 ```
 
 Due to the memory footprint restriction, the stored slow query records count is limited. If the specified `N` is greater than the records count, the returned records count may be smaller than `N`.
+
+The table below specifies the details of the output:
+
+| Column name | Description |
+|:------|:---- |
+| start | The starting time of SQL execution |
```

Comment:
```suggestion
| start | The starting time of the SQL execution |
```

Reply:
@TomShawn `the` here is not a must. Let's just keep one `the` in this phrase to make it sound simple.

Reply:
OK.